x-pack/legacy/plugins/lens/server/usage/types.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface LensUsage {
+  visualization_types_overall: Record<string, number>;
```

Comment:
Is there a reason you're using snake_case instead of camelCase? JavaScript is generally camel cased.

Reply:
This is the pattern being used in the rest of the metrics we collect